### DISCUSSION TITLE
Running .Net 8 on RHEL 7.1+

### DISCUSSION THREAD

**Discussion by kashti3** [dotnet/runtime#105825] [Q&A] [answered]
Hi, 
We are developing a .Net 8 application and have some customer who trying to run it over RHEL 7.1+.
I know that Microsoft is not officially support this OS type but this is a way to allow it? (the customer does not want to upgrade to new RHEL version)

May be I can installed some packages to allow the run over RHEL 7.1+

**Comment by huoyaoyuan** [comment]
.NET is targeting much higher version of glibc than old distros. [.NET 8](https://github.com/dotnet/core/blob/main/release-notes/8.0/supported-os.md#linux-compatibility) requires glibc 2.23+.

**Reply by kashti3** [reply]
I know and when trying to  run my application I get an error but I`m asking if there is a way to install them manually or some workaround to be able to run properly my application

**Reply by huoyaoyuan** [reply, marked as answer]
Updating glibc would have similar impact than updating OS.
I'm not familiar with library loader for Linux and not sure whether app private glibc will ever work.
